docs/sdk-and-tools/indices/validators.md

```diff
@@ -0,0 +1,29 @@
+---
+id: validators
```

Comment:
as discussed, rename the `id` to `es-index-validators`

Reply:
refactored.